template.rb

```diff
@@ -0,0 +1,7 @@
+# Run `rails new my-app -m https://raw.githubusercontent.com/bradgessler/litestack/master/template.rb`
+# to create a new Rails app with Litestack pre-installed.
+gem "litestack", github: "bradgessler/litestack"
```

Comment:
```suggestion
gem "litestack", github: "oldmoe/litestack"
```

Just as a reminder to do it before submitting for merging